src/components/visualizations/chart/chartOptions/test/comboChartOptions.spec.ts

```diff
@@ -36,3 +37,50 @@ describe("getComboChartSeries", () => {
         },
     );
 });
+
+describe("getComboChartStackingConfig", () => {
+    it("should return null when there is no stacking config", () => {
+        expect(getComboChartStackingConfig({}, [], null)).toBe(null);
+    });
+
+    it("should return default 'percent' stack value", () => {
+        expect(
+            getComboChartStackingConfig(
+                { stackMeasures: true },
+                [
+                    {
+                        yAxis: 0,
+                        type: "column",
```

Comment:
Using the constant here would be better. 
Could you please apply it here and the similar places

Reply:
done